src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -2980,6 +2988,51 @@ class ArnoldRenderer final : public ArnoldRendererBase
 
 	private :
 
+		void updateCameraMeshes()
+		{
+			for( const auto &it : m_globals->cameraMap() )
+			{
+				IECoreScene::ConstCameraPtr cortexCamera = it.second;
+				auto i = cortexCamera->parameters().find( "mesh" );
```

Comment:
This could be much simpler. Either use `parametersData()->member<StringData>( "mesh" )` if you genuinely want to silently ignore parameters with the wrong type, or use `parameter( cortexCamera->parameters(), "mesh", "" )` to get reporting for unexpected types - I think the latter is probably better?